src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.css

```diff
@@ -127,6 +127,10 @@
   align-items: center;
 }
 
+.breakpoint-panel .form-row.invalid .CodeMirror {
+  border-color: #dc75c5;
```

Comment:
If it's an error let's stick with some standard error color. The console ones are nice and not firetruck red.

https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/themes/webconsole.css#L31-L32

Reply:
went with `--red-50`